karma-base.conf.js

```diff
@@ -40,7 +40,7 @@ module.exports = function(config) {
 
     logLevel: config.LOG_INFO,
 
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Please keep this as `Chrome`, we can add change the browsers to `ChromeHeadless` in `karma-build-jasmine-chrome.conf.js` because we need to test both `headless` and `normal` chrome.